### ISSUE TITLE
Axis to buttons, wrong assignment in Rz-axis section.

### ISSUE THREAD

**Issue by Stout001** [FreeJoy-Team/FreeJoyConfiguratorQt#37]
I set up a potentiometer connected to the Rz-axis. From there I assign three buttons. In FJC everything seems to be all right, but when I test in VKB Button Tester I don't get the last button triggered. Only the first two.

If I do the same with Ry-axis everything is working and I get all three triggered.

Is this axis dependent as a "feature" or is it a bug?

When I'm on it with axes. Is it possible to assign two buttons and have a dead zone between them? The reason for this is that in IL-2 sturmovik when assigning two buttons they stop reacting when releasing the button. But in Freejoy they don't actually release, they shift from button 1 to button 2 and vice versa, so there's no stop in between. 

I tried different configurations using three buttons and decreasing the the triggering distance and it work, some sort, but not optimal as the buttons are triggered in a row inspite I have set the axis to center.
I also tried to assign the button numbers to the section with button 1, 2 and 3 and set the buttons to down, center and up. But the response is failing.

So in simple terms. If I'm in IL-2 sturmovik and want to assign a double commando to "Adjustable stabilizer axis" how do I do that so when joggling up the pitch raises, joggle to center the pitching stop and moving the joggle down the pitch is lowering?

It's quite easy to do with Joystick Gremlin, but I don't use that application anymore and that was the reason for changing from MMJOy2, Vjoy, Joystick Gremlin to just use Freejoy without external applications.
